contracts/asset-proxy/test/proxies.ts

```diff
@@ -30,6 +18,12 @@ import * as _ from 'lodash';
 
 import {
     artifacts,
+    DummyERC20TokenContract,
+    DummyERC20TokenTransferEventArgs,
+    DummyERC721ReceiverContract,
+    DummyERC721TokenContract,
+    DummyMultipleReturnERC20TokenContract,
+    DummyNoReturnERC20TokenContract,
```

Comment:
Can we not import from `../src`? Can we import from the exact file? It makes it harder to grep for when a directory is renamed...

Reply:
Discussed offline